### PR TITLE
Update download page (id)

### PR DESCRIPTION
**Pull request by meisyal**
Changes:

* Add information on Ruby releases and branches as mentioned on #1522 
* Display preview release if one exists (see commit c0612c029f43a671a6e680e225d18375ad37d08a).

### CODE REVIEW

**Comment by meisyal** [comment]
cc: @gozali

File: id/downloads/index.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ salah satu kakas bantu pihak ketiga yang telah disebutkan sebelumnya. Itu mungki
   * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
     sha256: {{ release.sha256.gz }}{% endfor %}
 
+{% if site.data.downloads.preview %}
+* **Preview releases:**{% for version in site.data.downloads.preview %}{% assign release = site.data.releases | where: "version", version | first %}
```
Comment (kuntoaji) [inline id/downloads/index.md]: May I know why only first? I think it's possible we have more than one preview releases. Personally I prefer to use original code from [this commit](https://github.com/ruby/www.ruby-lang.org/commit/c0612c029f43a671a6e680e225d18375ad37d08a).

```
{% if site.data.downloads.preview %}
* **Preview releases:**{% for release in site.data.downloads.preview %}
  * [Ruby {{ release.version }}]({{ release.url.gz }})<br>
	    sha256: {{ release.sha256.gz }}{% endfor %}
{% endif %}
```

Comment (meisyal) [reply id/downloads/index.md]: I followed the latest version of `en` Download page.

The Download page shows the latest version of releases. If we want to know specific or older release, we can check the Releases page. Please, refer to #1685 why presents the first, @kuntoaji.

**Comment by meisyal** [comment]
Thanks @kuntoaji

**Pull request merged**